<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>141</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="widget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>291</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/icons/10.png</normaloff>:/images/icons/10.png</iconset>
     </attribute>
     <attribute name="label">
      <string>列表项操作</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="toolButton_init">
         <property name="text">
          <string>初始化列表</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_clear">
         <property name="text">
          <string>清除列表</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_insert">
         <property name="text">
          <string>插入项</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_add">
         <property name="text">
          <string>添加项</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_delete">
         <property name="text">
          <string>删除当前项</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>291</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/icons/10.png</normaloff>:/images/icons/10.png</iconset>
     </attribute>
     <attribute name="label">
      <string>列表排序</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBox_sort">
         <property name="text">
          <string>允许排序</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_ascend">
         <property name="text">
          <string>升序排序</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="arrowType">
          <enum>Qt::UpArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_descend">
         <property name="text">
          <string>降序排序</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="arrowType">
          <enum>Qt::DownArrow</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>291</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/icons/10.png</normaloff>:/images/icons/10.png</iconset>
     </attribute>
     <attribute name="label">
      <string>信号发射时机</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>136</width>
        <height>223</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QToolButton" name="toolButton_txt_clear">
         <property name="text">
          <string>清空文本</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="toolButton_empty_line">
         <property name="text">
          <string>添加空行</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>511</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>当前项变化</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_editable">
         <property name="text">
          <string>项可编辑</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="toolButton_menu">
         <property name="text">
          <string>功能</string>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>16</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::MenuButtonPopup</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_all">
         <property name="text">
          <string>全选</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_empty">
         <property name="text">
          <string>空选</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_inv">
         <property name="text">
          <string>反选</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="listWidget">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <item>
        <property name="text">
         <string>List1</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/icons/13.png</normaloff>:/images/icons/13.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>List2</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/icons/13.png</normaloff>:/images/icons/13.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_initializer"/>
   <addaction name="action_clear"/>
   <addaction name="action_insert"/>
   <addaction name="action_add"/>
   <addaction name="action_delete"/>
  </widget>
  <action name="action_initializer">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/11.png</normaloff>:/images/icons/11.png</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_clear">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/8.png</normaloff>:/images/icons/8.png</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
  </action>
  <action name="action_insert">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/10.png</normaloff>:/images/icons/10.png</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
  </action>
  <action name="action_add">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/6.png</normaloff>:/images/icons/6.png</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/5.png</normaloff>:/images/icons/5.png</iconset>
   </property>
   <property name="text">
    <string>删除当前项</string>
   </property>
  </action>
  <action name="action_all">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="action_none">
   <property name="text">
    <string>空选</string>
   </property>
  </action>
  <action name="action_inv">
   <property name="text">
    <string>反选</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/3.png</normaloff>:/images/icons/3.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
